LibVLCSharp/Shared/Core.cs

```diff
@@ -79,15 +92,41 @@ static IntPtr PreloadNativeLibrary(string nativeLibrariesPath, string libraryNam
         {
             Debug.WriteLine($"Loading {libraryName}");
             var libraryPath = Path.Combine(nativeLibrariesPath, libraryName);
-            if(!File.Exists(libraryPath))
+
+#if !NETSTANDARD1_1
+            if (!File.Exists(libraryPath))
             {
                 Debug.WriteLine($"Cannot find {libraryPath}");
                 return IntPtr.Zero;
             }
-
+#endif
             return Native.LoadLibrary(libraryPath);// TODO: cross-platform load
         }
+
+        static bool IsWindows
+        {
+            get
+            {
+#if NET40
+            return Environment.OSVersion.Platform != PlatformID.MacOSX 
+                && Environment.OSVersion.Platform != PlatformID.Unix;
```

Comment:
Isn't `Win32NT` enough here? Do you count Xbox as "Windows" ? Do we really need to care about win95 and win98?

Reply:
Source : https://docs.microsoft.com/en-us/dotnet/api/system.platformid?view=netframework-4.7.2